src/platforms/android/common/performance/included-instrumentation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Included Instrumentation
+sidebar_order: 21
+description: "Learn what transactions are captured after tracing is enabled."
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+<Note>
+
+This is only available since Sentry's Android SDK version `4.3.0`
+
+</Note>
+
+### Android's Activity Instrumentation
+
+The Activity's instrumentation, once enabled, captures transactions for each Activity's start up, Transaction names follow pattern `ActivityName`, for example for an Activity called `MainActivity`, The Transaction's name is `MainActivity` and its operation is `navigation`.
+
+The transaction starts before each Activity's `onCreate` method is called.
+
+_The Activity's instrumentation is enabled by default, but you may disable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.activity.enable" android:value="false" />
+</manifest>
+```
```

Comment:
`l`: Shouldn't we also mention how to disable this with code?

Reply:
I'd keep it rather short as it's enabled by default, we already have the basic options page for both ways of doing it, which I've not added yet